<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideshowPanel</class>
 <widget class="QWidget" name="SlideshowPanel">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ComputerMonitoringWidget" name="monitoringWidget"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QToolButton" name="showPreviousButton">
        <property name="toolTip">
         <string>Previous</string>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="icon">
         <iconset resource="../../core/resources/core.qrc">
          <normaloff>:/core/go-previous.png</normaloff>:/core/go-previous.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="startStopButton">
        <property name="toolTip">
         <string>Start/pause</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/master.qrc">
          <normaloff>:/master/media-playback-start.png</normaloff>
          <normalon>:/master/media-playback-pause.png</normalon>:/master/media-playback-start.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="showNextButton">
        <property name="toolTip">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="../../core/resources/core.qrc">
          <normaloff>:/core/go-next.png</normaloff>:/core/go-next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="durationSlider">
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>1000</number>
        </property>
        <property name="value">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComputerMonitoringWidget</class>
   <extends>QListView</extends>
   <header>ComputerMonitoringWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../core/resources/core.qrc"/>
  <include location="../resources/master.qrc"/>
 </resources>
 <connections/>
</ui>
